test/calculator.cc

```diff
@@ -0,0 +1,39 @@
+/*
```

Comment:
I discussed about this test with @hwanseung but we thought that this test might not be necessary.
Because the calculator example is already tested by Mocha with JS.
But anyway, we need an test example for unittest as well.
So, I think we might have two options as follows:
  - Remove this test and rewrite a new test we really need (e.g. unit test for NativeTypeTraits)
  - Keep this test as-is for now.

The first option is good to me but I also don't want this PR to be pending.
So, the second option is also good to me.

WDYT? @hwanseung @yjaeseok

Reply:
I choose to remove this test and rewrite a new test we really need 
But I am having trouble adding new test code right now.

So I would like to add only testing framework at this PR.

Reply:
@yjaeseok okay, it's a good idea. (and Congratulation on the birth of junior Jake!)

Reply:
But....... If I do this, this PR becomes for .gitmodules only. T.T

Reply:
So I will make the new test ASAP!

Reply:
@yjaeseok Ah, sorry for confusing.
I meant that we might not need calculator test.
Your CMakeList.txt and main() is still needed I thought.
But you can do it in second patch, which do you prefer?